Sources/FoundationExtensions/Decoder+Extensions.swift

```diff
@@ -77,9 +77,48 @@ extension DefaultValue: Encodable where Value: Encodable {
 
 }
 
+// MARK: - IgnoreEncodable
+
+/// A property wrapper that allows not encoding a value.
+/// - Example:
+/// ```
+/// struct Data {
+///     @IgnoreEncodable var data: String // this value won't be encoded
+/// }
+/// ```
+@propertyWrapper
+struct IgnoreEncodable<Value> {
+
+    var wrappedValue: Value
+
+}
+
+extension IgnoreEncodable: Decodable where Value: Decodable {
+
+    init(from decoder: Decoder) throws {
+        self.wrappedValue = try decoder.singleValueContainer().decode(Value.self)
+    }
+
+}
+
+extension IgnoreEncodable: Encodable {
```

Comment:
A transparent benefit of this is that `@IgnoreEncodable` conforms to `Encodable` without relying on `Value` (because it doesn't actually do anything). Which means that it can be used for types that aren't `Encodable` (like `[String: Any]`):
```swift
struct Data: Encodable {
  @IgnoreEncodable var rawData: [String: Any] // this compiles!
}
```